src/simulate/README.md

```diff
@@ -7,9 +7,54 @@
 3. To install the required modules for the utility do `pip install -r requirements.txt`
 
 ### Run 
- `python3 main.py <path-to properties-file> <path-to-json-events-directory>` 
+ The last argument if not provided default to 1 run
+ `python3 main.py <path-to properties-file> <path-to-json-events-directory> <number-of-times-to-run-Json-Events>` 
+
+
+###### Docker local Run
+ First a docker build needed and then running the build in container. 
+ If the build image changes a new name needs to be given like `simulator1` or delete the old build images to use the same name
+1. Build the images first  ` docker build -t simulator .` 
+2. Run image in a container `docker run -it --rm --name simulator-run simulator`
+ 
+
+##### Useful Docker Commands 
```

Comment:
These are generic instructions.  It would be better to put these in the Wiki.  Otherwise they need to be in all projects READMEs and some will get out of date.

Reply:
I have added them to Wiki page saying Docker tips & Tricks and removed from here